packages/dm-core-plugins/src/grid/GridPlugin.tsx

```diff
@@ -52,7 +52,10 @@ export const GridPlugin = (
   }
 
   return (
-    <Grid className='dm-parent-plugin' {...internalConfig.size}>
+    <Grid
+      className='dm-plugin-padding dm-parent-plugin'
```

Comment:
Are these classNames used without tailwind?

Reply:
Yes, these are minor classes that help with us with the padding that we've discussed earlier 🙂 It's styled-components global styles and class names that don't affect anything else, so still no configuration for users